.github/workflows/workspace.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       - uses: actions/checkout@v1
       - uses: actions-rs/toolchain@v1
         with:
-          toolchain: 1.41.0 # MSRV
+          toolchain: 1.47.0 # MSRV
```

Comment:
I think we can remove the "MSRV" comment from here, since it's not about MSRV, but about pinning clippy version.